bokehjs/test/unit/models/formatters/datetime_tick_formatter.ts

```diff
@@ -223,6 +223,13 @@ describe("DatetimeTickFormatter", () => {
       expect(labels).to.be.equal(["0ms", "5ms", "10ms"])
     })
   })
+  describe("scale_up_boundary", () => {
+    it("should handle boolean", () => {
```

Comment:
I think this test needs a new name, up above there was a "should handle boolean" because `strip_leading_zeros` can accept boolean values and numeric values, and that test checked that the boolean case worked. But here `scale_up_boundary` is boolean flag, it _only_ accepts booleans in anycase. This test is checking one of the two cases specfically.

Reply:
Agreed. I will update this as soon as we agreed on a suitable property name.